extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -322,6 +322,7 @@ public static function onSkinAfterBottomScripts( $skin, &$bottomScriptsText ) {
         console.log("canonical is: " + changePay.get_canonical_url());
     },
     paid: function(status_code, msg) {
+	$('wikia-ad').hide();
```

Comment:
@BladeBronson I'm going to close this PR since the change is inside the code that is moved to changePay-wikia.js